pkg/ddevapp/webserver_config_assets/nginx-site-backdrop.conf

```diff
@@ -1,4 +1,6 @@
 # ddev backdrop config
+# https://www.drupal.org/project/drupal/issues/2937161
+# https://www.drupal.org/project/drupal/issues/3336659
```

Comment:
Backdrop is probably not similar enough to Drupal8+ to deserve these links.  DDEV's drupal7 config is based on time-tested situation like backdrop.

Reply:
```suggestion
```